docs/community/publications.md

```diff
@@ -102,6 +102,32 @@ If you write a paper using results obtained with the help of
 
 Older releases are announced in the following preprints:
```

Comment:
I know this is not something you have modified, but theoretically the paper you cite below that line is not a preprint but is actually a published article. Would we want to change that line to say more something along the line of : Older releases are announced in the following articles:

```suggestion
Older releases are announced in the following articles:
```
?

Reply:
Yes, that sounds reasonable. I think the text dates back to a time when we only had preprints.